data/datafiles.json

```diff
@@ -640,6 +640,13 @@
 	    "filename": "ramses_extra_fields_small",
 	    "size": "15 MB",
 	    "url": "http://yt-project.org/data/ramses_extra_fields_small.tar.gz"
+	},
+	{
+	    "code": "RAMSES",
+	    "description": "Hydro run with stars and black holes (sink).",
+	    "filename": "ramses_sink_00016",
+	    "size": "7.2 MB",
```

Comment:
The file I downloaded from [PR 1548](https://github.com/yt-project/yt/files/1284513/ramses_sink_00016.tar.gz) has 3.9MB. Is it the same data?

Reply:
My bad, I'm fixing the size here. Should I mention the unzipped size or the file size?

Reply:
Zipped

Reply:
the file size

Reply:
Done.